<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgUserCardEdit</class>
 <widget class="QDialog" name="dlgUserCardEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblBarCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Barcode: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledBarCode">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledComment">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblUserCardType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Usercard type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbUserCardType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>250</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblUnits">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Units:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledUnits">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblValidDate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Valid till:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtpValidDate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPinCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pin code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledPinCode">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="btbButtons">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>dlgUserCardEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>dlgUserCardEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
